tests/unit/test_configuration.py

```diff
@@ -544,3 +521,47 @@ def test_no_noise():
             column_noise_dict = dataset_column_dict[column]
             for column_noise_type in column_noise_dict.keys():
                 assert column_noise_dict[column_noise_type][Keys.CELL_PROBABILITY] == 0.0
+
+
+@pytest.mark.parametrize(
+    "column, noise_type, noise_level",
+    [
+        ("age", "copy_from_household_member", 0.2),
+        ("age", "copy_from_household_member", 0.95),
+        ("first_name", "use_nickname", 0.05),
+        ("first_name", "use_nickname", 0.85),
+        ("date_of_birth", "copy_from_household_member", 0.15),
+        ("date_of_birth", "copy_from_household_member", 0.90),
+    ],
+)
+def test_validate_noise_level_proportions(caplog, column, noise_type, noise_level):
+    """
```

Comment:
I was very happy I could parametrize with caplog fixture here but it does seem weird that I do not need to clear the records if it is a fixture.